README.md

```diff
@@ -20,6 +21,11 @@ For the driver, change the library name in privateExportsConfigs, the driver nam
 For the maven artifact names, those are all in publish.gradle about 40 lines down.
 
 ## Building and editing
+
 This uses gradle, and uses the same base setup as a standard GradleRIO robot project. This means you build with `./gradlew build`, and can install the native toolchain with `./gradlew installRoboRIOToolchain`. If you open this project in VS Code with the wpilib extension installed, you will get intellisense set up for both C++ and Java.
 
-By default, this template builds against the latest WPILib development build. To build against the last WPILib tagged release, build with `./gradlew build -PreleaseMode`.
\ No newline at end of file
+By default, this template builds against the latest WPILib development build (`wpi.maven.useDevelopment` is implicitly set to true). To build against the last WPILib tagged release, build with `./gradlew build -PreleaseMode`.
+
+### Post-season builds
+
+To fix builds after WPILib releases an alpha version for the next year, set `wpi.maven.useDevelopment = false` or increment the year in the `year.+` version by one.
```

Comment:
This is not correct. wpi.maven.useDevelopment is not a thing in a non gradlerio scope. The way to fix would be to change the releaseMode flag.